folly/experimental/symbolizer/Dwarf.cpp

```diff
@@ -477,41 +481,99 @@ bool Dwarf::findLocation(
   folly::StringPiece compilationDirectory;
   folly::Optional<folly::StringPiece> mainFileName;
   folly::Optional<uint64_t> baseAddrCU;
+  folly::Optional<uint64_t> rangesOffset;
+  bool seenLowPC = false;
+  bool seenHighPC = false;
+  enum : unsigned {
+    kStmtList = 1U << 0,
+    kCompDir = 1U << 1,
+    kName = 1U << 2,
+    kLowPC = 1U << 3,
+    kHighPCOrRanges = 1U << 4,
+  };
+  unsigned expectedAttributes =
+    (kStmtList | kCompDir | kName | kLowPC | kHighPCOrRanges);
```

Comment:
kHighPCORanges isn't needed here

Reply:
I thought I'd already fixed this. The intent was clearly there :)